Server/Ebenezer/User.cpp

```diff
@@ -12127,6 +12202,26 @@ void CUser::SendNpcSay(const EXEC* pExec)
 	Send(send_buff, send_index);
 }
 
+void CUser::SendSay(short m1, short m2, short m3, short m4, short m5, short m6, short m7, short m8, short m9, short m10)
```

Comment:
I don't really like these names. I'm assuming it's 'm' for message, but even if that's the case, that's not actually true for the first 2; the first 2 are event IDs (which it will forward back to the server in its response upon clicking), for the 'up' button and the 'ok' button consecutively.

At the very least we should ensure these are named separately so they don't just look like arbitrary message/text IDs.